src/libsync/networkjobs.cpp

```diff
@@ -477,7 +477,8 @@ bool CheckServerJob::finished()
         if( status.contains("installed")
                 && status.contains("version")
                 && status.contains("versionstring") ) {
-
+            // Newer servers don't disclose anything in that empty string "version" anymore
```

Comment:
The server still sends "version" and "versionstring", but it will be empty. I had to look this up too. Expand comment?